show_plot.py

```diff
@@ -150,26 +124,26 @@ def _update_plot():
     crop_image = True
     img = read_image(img_name)
     img = img[::-1, :]
-    print("Uploading image...")
+    print("[PLOT] Uploading image...")
     try:
         y = predict(img_name, verbose=True)
     except:
```

Comment:
This is a bare try/except block. I'd prefer

``` python
try:
    y = predict(...)
except FaceNotFoundException:
    ...
```

If the joblib file can't be read, this will make that more obvious.

Reply:
There's even a pylint warning on using a bare try/except: http://pylint-messages.wikidot.com/messages:w0702